config/string.c

```diff
@@ -186,6 +186,40 @@ static intptr_t string_native_get(const struct ConfigSet *cs, void *var,
   return (intptr_t) str;
 }
 
+/**
+ * string_string_plus_equals - Concat String to a string - Implements ConfigSetType::string_plus_equals
+ */
+static int string_string_plus_equals(const struct ConfigSet *cs, void *var, const struct ConfigDef *cdef,
+                             const char *value, struct Buffer *err)
+{
+  /* Skip if the value is missing or empty string*/
+  if (!value || (value && (value[0] == '\0')))
+    return CSR_SUCCESS | CSR_SUC_NO_CHANGE;
+
+  int rc = CSR_SUCCESS;
+
+  if (var)
```

Comment:
We don't need this check as `var` is guaranteed in the contract for `string_plus_equals()`:
https://github.com/neomutt/neomutt/blob/758d453f0326ee1e8f6b69931277820117e62c88/config/set.h#L159-L173